dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -711,8 +804,8 @@ def prepare_provider_packages(
         sys.exit(0)
     get_console().print("\n[success]Successfully built packages!\n\n")
     get_console().print("\n[info]Packages available in dist:\n")
-    for file in sorted(DIST_DIR.glob("apache*")):
-        get_console().print(file.name)
+    for dist_info in _dist_packages(package_format=package_format, build_type="providers"):
+        get_console().print(str(dist_info))
```

Comment:
For providers I do not add this check but maybe I should?
In case of `apache-airflow` it doesn't big overhead, build package from sources take for about 1m - 1m 20s, and additional 10 seconds in negligible in the other hand create new venv for each provider package might add huge overhead in total.

Reply:
I think for providers it's less of a problem, because 1) we do not have custom build hooks there 2) pyproject.toml is automatically generated from template, and is basically the same for every provided (- paths it uses). And we also check it during main build. But if you feel like adding it would be good - we can definitely add it as follow-up

Reply:
BTW. We could create single venv and install all providers there - after all of them are prepared, we usually prepare 10-15 providers together, so I think the overhead will be almost not noticeable there.

Reply:
BTW. Since it is sdist - it is only enabled in canary build, so you can either change the condtion in ci.yml it to test it in CI or push it to "airflow" repo and assign `canary` label to get it work in the same way as canary build.

Reply:
Yeah, will push it into the airflow repo tomorrow, after check how it impact on providers.